packages/databyss-ui/theming/views.js

```diff
@@ -56,6 +56,15 @@ const widthVariants = {
   form: {
     maxWidth: pxUnits(550),
   },
+  dropdownMenuSm: {
```

Comment:
Despite the abbreviations in styled-system, let's use full names ("Medium", "Large", etc)...